proxy/server/manager.go

```diff
@@ -129,6 +129,7 @@ type Manager struct {
 	namespaces  [2]*NamespaceManager
 	users       [2]*UserManager
 	statistics  *StatisticManager
+	EncryptKey  string
```

Comment:
Is it better to move EncryptKey to struct Server?

Reply:
Yeah,i'm afraid we will use EncryptKey in manager, but it's ok now, i will change it.